checkstyle-suppressions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "suppressions_1_1.dtd">
+
+<suppressions>
+    <suppress files="[\\/]src[\\/]test[\\/].*" id="MethodNameRegular" />
+    <suppress files="[\\/]src[\\/]main[\\/].*" id="MethodNameTest" />
```

Comment:
@nivancevic This should work and it works for files in src/main folder (when you add '_" to method name it will fail).

However it is not working for test files and I cannot figure out why. Idea is to have two rules, one for test and one for production code and in suppression file exclude test from production and vice versa.